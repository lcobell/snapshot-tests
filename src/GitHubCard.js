import Card from "react-bootstrap/Card";
import Button from "react-bootstrap/Button";
import renderer from "react-test-renderer";

const GitHubCard = () => {
  return (
    <Card style={{ width: "18rem" }}>
      <Card.Img variant="top" src="./logan.jpg" />
      <Card.Body>
        <Card.Title>Logan CoBell</Card.Title>
        <h3>Find me on GitHub at github.com/lcobell</h3>
        <Card.Text>
          My name is Logan CoBell and I am a Software Development student at Old
          Dominion University and Discovery Partners Institute. To fund this
          endeavor I bartend at RPM Steak in Chicago, Illinois. In my free time
          I enjoy running with my dogs, working out, and checking out Chicago's
          restaurant scene.
        </Card.Text>
        <Button variant="primary">Go somewhere</Button>
      </Card.Body>
    </Card>
  );
};

export default GitHubCard;
