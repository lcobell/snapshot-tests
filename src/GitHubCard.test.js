import { render, screen } from "@testing-library/react";
import App from "./App";
import renderer from "react-test-renderer";
import GitHubCard from "./GitHubCard";
test("renders a card", () => {
  const tree = renderer.create(<GitHubCard />).toJSON();
  expect(tree).toMatchSnapshot();
});

// const GitHubCard = () => {
//   return (
//     <Card style={{ width: "18rem" }}>
//       <Card.Img variant="top" src="./logan.jpg" />
//       <Card.Body>
//         <Card.Title>Logan CoBell</Card.Title>
//         <h3>Find me on GitHub at github.com/lcobell</h3>
//         <Card.Text>
//           My name is Logan CoBell and I am a Software Development student at Old
//           Dominion University and Discovery Partners Institute. To fund this
//           endeavor I bartend at RPM Steak in Chicago, Illinois. In my free time
//           I enjoy running with my dogs, working out, and checking out Chicago's
//           restaurant scene.
//         </Card.Text>
//         <Button variant="primary">Go somewhere</Button>
//       </Card.Body>
//     </Card>
//   );
// };

export default GitHubCard;
