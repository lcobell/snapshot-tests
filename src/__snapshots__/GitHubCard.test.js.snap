// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a card 1`] = `
<div
  className="card"
  style={
    Object {
      "width": "18rem",
    }
  }
>
  <img
    className="card-img-top"
    src="./logan.jpg"
  />
  <div
    className="card-body"
  >
    <div
      className="card-title h5"
    >
      Logan CoBell
    </div>
    <h3>
      Find me on GitHub at github.com/lcobell
    </h3>
    <p
      className="card-text"
    >
      My name is Logan CoBell and I am a Software Development student at Old Dominion University and Discovery Partners Institute. To fund this endeavor I bartend at RPM Steak in Chicago, Illinois. In my free time I enjoy running with my dogs, working out, and checking out Chicago's restaurant scene.
    </p>
    <button
      className="btn btn-primary"
      disabled={false}
      type="button"
    >
      Go somewhere
    </button>
  </div>
</div>
`;
